docs/dev/framework/templates/getting-started.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "1. Getting started"
```

Comment:
```suggestion
title: "1. Getting started"
```

The numbering is not required and we do not use that anywhere else. You can use the `weight` property, if you want to order the articles in a specific way.